tests/core/analytics/analyticsreporter.js

```diff
@@ -22,9 +21,12 @@ describe('reporting events', () => {
   });
 
   it('throws an error if given a non-AnalyticsEvent', () => {
-    expect(() => {
-      analyticsReporter.report({ event_type: 'fake event' });
-    }).toThrow(AnswersAnalyticsError);
+    const consoleErrorSpy = jest.spyOn(global.console, 'error').mockImplementation();
```

Comment:
can this be simplified to `jest.spyOn(console, 'error');`?

Reply:
I can remove `global.` but I kind of prefer to mock implementation of console so it doesn't actually log these errors in the terminal

Reply:
I usually prefer to mock implementation of console so it doesn't actually console log the error messages in terminal. But it doesn't look like it does here in SDK so I can remove